### PR TITLE
Add nms and batched_nms doc

### PR DESCRIPTION
**Pull request by RichardWooSJTU**
Add python api nms and batched_nms doc

### CODE REVIEW

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.. _cn_api_paddle_vision_ops_nms:
+
+nms
+-------------------------------
+
+.. py:function:: paddle.vision.ops.nms(boxes, iou_threshold=0.3, scores=None, category_idxs=None, categories=None,top_k=None)
```
Comment (wadefelix) [inline docs/api/paddle/vision/ops/nms_cn.rst]: need a space before `topk=None`

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+返回
+:::::::::
+    - Tensor - 被NMS保留的检测边界框的索引，它应当是一个形状为(num_boxes)的1-D Tensor。
+
+
+代码示例
+:::::::::
+
+..  code-block:: python
```
Comment (wadefelix) [inline docs/api/paddle/vision/ops/nms_cn.rst]: if the `code-block` is same as the code in the function's docstring, the `code-block` here can be replaced by `COPY-FROM: paddle.vision.ops.nms` OR `COPY-FROM: paddle.vision.ops.nms:<code-block name>`

File: docs/api/paddle/vision/ops/nms_cn.rst (hunk 1 above)
Comment (RichardWooSJTU) [reply docs/api/paddle/vision/ops/nms_cn.rst]: fixed

File: docs/api/paddle/vision/ops/nms_cn.rst (hunk 2 above)
Comment (RichardWooSJTU) [reply docs/api/paddle/vision/ops/nms_cn.rst]: done~

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.. _cn_api_paddle_vision_ops_nms:
+
+nms
+-------------------------------
+
+.. py:function:: paddle.vision.ops.nms(boxes, iou_threshold=0.3, scores=None, category_idxs=None, categories=None, top_k=None)
+
+非极大抑制(non-maximum suppression, NMS)用于在目标检测应用对检测边界框(bounding box)中搜索局部最大值，即只保留处于同一检测目标位置处重叠的框中分数最大的一个框。IoU(Intersection Over Union) 被用于判断两个框是否重叠，该值大于门限值则被认为两个框重叠。其计算公式如下：
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst]: 门限值 -> 门限值(iou_threshold)

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    IoU = \frac{intersection\_area(box1, box2)}{union\_area(box1, box2)}
+
+如果参数scores不为None，输入的boxes会首先按照它们对应的score降序排序，否则将默认输入的boxes为排好序的。
+如果category_idxs和categories不为None，分类NMS将会被执行，也就是说，nms过程会在每一个类别的框当中分别进行计算，计算结果会被组合起来然后按照得分倒序排列。
+如果top_k不为None的话，排序的计算结果中仅有前k个元素会被返回，否则会返回所有的元素。
+
+参数
+:::::::::
+    - boxes(Tensor) - 待进行计算的框坐标，它应当是一个形状为(num_boxes, 4)的2-D Tensor，以[[x1, y1, x2, y2], ...]的形式给出。其中(x1, y1)是左上角的坐标值，(x2, y2)是右下角的坐标值。数据类型可以是float32或float64.请注意，nms无需传入score信息，因此请提前按照框的score对boxes进行逆序排列。
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst]: Missing this paragraph ``Their relation should be 0 <= x1 < x2 && 0 <= y1 < y2.``

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    - iou_threshold(float32) - 用于判断两个框是否重叠的IoU门限值。 如果IoU(box1, box2) > threshold， box1和box2将被认为是重叠框。
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst]: Chinese Document：
float32 -> float32，可选；
Missing Default value of iou_threshold

English Document：
float32 -> float32, optional；

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    - scores(Tensor，可选) - 与boxes参数对应的score，它应当是一个形状为(num_boxes)的1-D Tensor。数据类型可以是float32或float64。
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst]: (num_boxes) -> [num_boxes]

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    - category_idxs(Tensor，可选) - 与boxes参数对应的类别编号，它应当是一个形状为(num_boxes)的1-D Tensor。数据类型为int64。
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst]: As above.

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    - categories(List，可选) - 类别列表，它的每个元素应该是唯一的，满足categories == paddle.unique(class_idxs).
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst]: Missing default values for both Chinese and English documents

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    - top_k(int64，可选) - 需要返回的分数最高的boxes索引数量。该值须小于等于num_boxes。
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst]: As above.

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 10]
```diff
@@ -0,0 +1,35 @@
+.. _cn_api_paddle_vision_ops_nms:
+
+nms
+-------------------------------
+
+.. py:function:: paddle.vision.ops.nms(boxes, iou_threshold=0.3, scores=None, category_idxs=None, categories=None, top_k=None)
+
+非极大抑制(non-maximum suppression, NMS)用于在目标检测应用对检测边界框(bounding box)中搜索局部最大值，即只保留处于同一检测目标位置处重叠的框中分数最大的一个框。IoU(Intersection Over Union) 被用于判断两个框是否重叠，该值大于门限值(iou_threshold)则被认为两个框重叠。其计算公式如下：
```
Comment (RichardWooSJTU) [inline docs/api/paddle/vision/ops/nms_cn.rst:8]: Finish "门限值 -> 门限值(iou_threshold)"

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+如果top_k不为None的话，排序的计算结果中仅有前k个元素会被返回，否则会返回所有的元素。
+
+参数
+:::::::::
+    - boxes(Tensor) - 待进行计算的框坐标，它应当是一个形状为[num_boxes, 4]的2-D Tensor，以[[x1, y1, x2, y2], ...]的形式给出，数据类型可以是float32或float64，其中(x1, y1)是左上角的坐标值，(x2, y2)是右下角的坐标值，其关系应符合``0 <= x1 < x2 && 0 <= y1 < y2``。
+    - iou_threshold(float32，可选) - 用于判断两个框是否重叠的IoU门限值。 如果IoU(box1, box2) > threshold， box1和box2将被认为是重叠框。默认为：0.3。
+    - scores(Tensor，可选) - 与boxes参数对应的score，它应当是一个形状为[num_boxes]的1-D Tensor。数据类型可以是float32或float64。默认为：None。
+    - category_idxs(Tensor，可选) - 与boxes参数对应的类别编号，它应当是一个形状为[num_boxes]的1-D Tensor。数据类型为int64。默认为：None。
+    - categories(List，可选) - 类别列表，它的每个元素应该是唯一的，满足categories == paddle.unique(class_idxs)。默认为：None。
+    - top_k(int64，可选) - 需要返回的分数最高的boxes索引数量。该值须小于等于num_boxes。默认为：None。
```
Comment (RichardWooSJTU) [inline docs/api/paddle/vision/ops/nms_cn.rst:27]: - Finish unifying English and Chinese version.
- Finish adding default value.

File: docs/api/paddle/vision/ops/nms_cn.rst [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+.. py:function:: paddle.vision.ops.nms(boxes, iou_threshold=0.3, scores=None, category_idxs=None, categories=None, top_k=None)
+
+非极大抑制(non-maximum suppression, NMS)用于在目标检测应用对检测边界框(bounding box)中搜索局部最大值，即只保留处于同一检测目标位置处重叠的框中分数最大的一个框。IoU(Intersection Over Union) 被用于判断两个框是否重叠，该值大于门限值(iou_threshold)则被认为两个框重叠。其计算公式如下：
+
+.. math:: 
+
+    IoU = \frac{intersection\_area(box1, box2)}{union\_area(box1, box2)}
+
+如果参数scores不为None，输入的boxes会首先按照它们对应的score降序排序，否则将默认输入的boxes为排好序的。
+如果category_idxs和categories不为None，分类NMS将会被执行，也就是说，nms过程会在每一个类别的框当中分别进行计算，计算结果会被组合起来然后按照得分倒序排列。
```
Comment (TCChenlong) [inline docs/api/paddle/vision/ops/nms_cn.rst:16]: Add a blank line before Line 15 & Line 16.
image

Comment (RichardWooSJTU) [reply docs/api/paddle/vision/ops/nms_cn.rst:16]: done~

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Pull request merged**